.gitignore

```diff
@@ -0,0 +1,4 @@
+react-widgets/node_modules
+react-widgets/bundle.js
+react-widgets/bundle.js.map
+.DS_Store
```

Comment:
I added .DS_Store to my gitignore because it is a mac specific private file and I want my git file to be as platform agnostic as possible